R/subset_datarqst.R

```diff
@@ -8,16 +8,28 @@
 #'
 #' When a request is made, `subset_datarqst` can be used
 #' to take a subset of this larger file to cover the specific
-#' populations/species requested by a user.
+#' populations/species requested by a user. The request can be specified
+#' either through the \code{PopID} and \code{Species} arguments, or through
+#' the \code{PopSpec_Filter} argument.
 #'
 #' @param file The location of the file in the standard format (.RDS file). This file is created using `run_pipelines`.
 #' @param PopID Character vector of three letter population codes. Include all populations that are requested.
 #' @param Species Character vector of six letter species codes. Include all species that are requested.
-#' @param filter Character vector of unique population species combinations (in the format PopID_Species).
-#' Include all unique population species combinations requested.
-#'
-#' @return Create 4 .csv files at the location of the standard data. Files will be given a unique number
-#' based on the date when the subset was performed.
+#' @param PopSpec_Filter Character vector of unique population species combinations (in the format PopID_Species).
```

Comment:
Being pendantic here, but if this one is called 'PopSpec_Filter' then shouldn't the others be 'PopID_Filter' and 'Species_Filter' 🤔. Maybe just 'PopSpec' would be better.

Reply:
Changed as suggested.